### ISSUE TITLE
Generico Filter for TinyMCE

### ISSUE THREAD

**Issue by JudithBirke** [justinhunt/moodle-atto_generico#6]
Hi Justin,
Could you please let me know whether you are planning to make Generico Filter also available for TinyMCE? Thanks.
Best wishes
Judith

**Comment by justinhunt** [comment]
Yes I am. I have really fallen behind on the TinyMCE thing. But the Generico filter should work regardless of the HTML editor. Its the icon on the HTML editor toolbar that makes it easy to select and build Generico filter strings that is not ready yet. You probably knew that. I just wanted to make it clear.

**Comment by JudithBirke** [comment]
Thanks, Justin, for the prompt response.
